pytransform3d/test/test_batch_rotations.py

```diff
@@ -42,6 +43,34 @@ def test_norm_vectors_zero():
     assert_array_almost_equal(V_unit, V)
 
 
+def test_norm_axis_angles():
+    rng = np.random.default_rng(843)
+    A = np.hstack((
+        pbr.norm_vectors(rng.standard_normal(size=(10, 3))),
+        np.pi + rng.uniform(size=(10, 1))
+    ))
```

Comment:
Since you base all your tests on `A`, consider a small refactoring for more clarity (after this the reader knows, "ah yeah those are multiple"):

```suggestion
    # create a batch of axis-angle instances
    num_instances = 10
    A = np.hstack((
        pbr.norm_vectors(rng.standard_normal(size=(num_instances, 3))),
        np.pi + rng.uniform(size=(num_instances, 1))
    ))
```